src/modules/isShopgateIosApp.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
If the amount of files would increase. I would start creating maybe a utility class to bring this together in cases were this fits but I think that one responsibility of a method/class should go to one file. This will give a clean and easy to understand structure

Reply:
After discussion with Carsten I will create one file for the device identification